Write a python program to import and export data using Pandas and show the details of the dataset like number of rows, columns, first five rows, size, number of missing values, sum, average, min and max values from the numerical columns.

In [1]:
import pandas as pd

In [2]:
df1 = pd.read_csv('Normal-ECG.csv')
df2 = pd.read_csv('Dia-ECG.csv')

In [3]:
mean_value = df1.mean()
max_value = df1.max()
min_value = df1.min()
std_dev = df1.std()
median_value = df1.median()

In [4]:
print("Mean value:\n", mean_value)
print("\nMax value:\n", max_value)
print("\nMin value:\n", min_value)
print("\nStandard Deviation:\n", std_dev)
print("\nMedian value:\n", median_value)

Mean value:
 6   -0.129339
dtype: float64

Max value:
 6    735
dtype: int64

Min value:
 6   -961
dtype: int64

Standard Deviation:
 6    89.986215
dtype: float64

Median value:
 6   -2.0
dtype: float64


In [5]:
df2.replace(['ekg', 'uv'], 0, inplace=True)

In [6]:
mean_value = df2.mean()
max_value = df2.max()
min_value = df2.min()
std_dev = df2.std()
median_value = df2.median()

In [7]:
print("Mean value:\n", mean_value)
print("\nMax value:\n", max_value)
print("\nMin value:\n", min_value)
print("\nStandard Deviation:\n", std_dev)
print("\nMedian value:\n", median_value)

Mean value:
 -38   -0.225438
dtype: float64

Max value:
 -38    584
dtype: int64

Min value:
 -38   -1741
dtype: int64

Standard Deviation:
 -38    66.430087
dtype: float64

Median value:
 -38    1.0
dtype: float64


## Comparison Table

| Statistic   | ECG         | Diabetes-ECG     |
|-------------|-------------|-------------|
| MEAN   | -0.129339    | -0.225438    |
| MEDIAN | -2          | 1           |
| **STD DEV** | **89.986215** | **66.430087** |
| MAX VALUE | 735        | 584         |
| MIN VALUE | -961       | -1741       |
